front_end/src/components/booking/Confirmation.js

```diff
@@ -1,50 +1,176 @@
+    createData("Subtotal", "$" + bookingData.subtotal.toFixed(2)),
+    createData(
+      "Cancellation Policy",
+      "Cancellations must be received 48 hours before check-in date for full refund"
+    )
+  ];
+
+  return (
+    <Grid
+      xs={isWidthDown("xs", width) ? 11 : 12}
+      container
+      spacing={isWidthDown("xs", width) ? 0 : 8}
+      justify="center"
+      style={{ margin: isWidthDown("xs", width) ? 10 : -8 }}
+    >
+      <Grid item xs={10}>
+        <Typography
+          variant="h4"
+          gutterBottom
+          align="center"
+          style={{ marginTop: 40 }}
+        >
+          {bookingData.hotelName}
+        </Typography>
+        <Typography variant="h6" gutterBottom align="center">
+          {bookingData.hotelAddress}
+        </Typography>
+      </Grid>
+
+      <Grid item xs={isWidthDown("xs", width) ? 11 : 5}>
+        <CardMedia
+          style={{ width: "100%", height: 200 }}
+          image={require("../landing_page/SF.jpg")}
+        />
+      </Grid>
+
+      <Grid item xs={isWidthDown("xs", width) ? 11 : 5}>
+        <CardMedia
+          style={
+            isWidthDown("xs", width)
+              ? { width: "100%", height: 200, marginTop: 20 }
+              : { width: "100%", height: 200 }
+          }
+          image={require("../landing_page/SD.jpg")}
+        />
+      </Grid>
+      <Grid item xs={10}>
+        <Typography
+          style={{
+            display: "flex",
+            justifyContent: "center",
+            marginBottom: 20
+          }}
+        >
+          {" "}
+          <br />
+          Dear Mr/Ms. {bookingData.Lastname},
+          <br />
+          Thank you for choosing {bookingData.hotelName}. It is our pleasure to
+          confirm the following reservation.
+        </Typography>
+      </Grid>
+      <Grid item xs={isWidthDown("xs", width) ? 11 : 4}>
+        <div
+          style={{
+            color: "white",
+            background: "linear-gradient(to right, #0c4b78, #3d4e96, #2c76a9)"
+          }}
+        >
+          {" "}
+          RESERVATION DETAILS
+        </div>
+        <Table>
+          <TableBody>
+            {ReservationRows.map(row => (
+              <TableRow
+                key={row.id}
+                style={{
+                  padding: 0
+                }}
+              >
+                <TableCell align="left" style={{ padding: 0, width: 150 }}>
+                  {row.firstCol}
+                </TableCell>
+                <TableCell align="left" style={{ padding: 0 }}>
+                  {row.secondCol}
+                </TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+      </Grid>
+
+      <Grid item xs={isWidthDown("xs", width) ? 11 : 4}>
+        <div
+          style={{
+            color: "white",
+            background: "linear-gradient(to right, #0c4b78, #3d4e96, #2c76a9)"
+          }}
+        >
+          {" "}
```

Comment:
there is a couple {" "}, are they needed?

Reply:
removed